frontend/sw360-portlet/src/main/webapp/html/components/includes/releases/detailOverview.jspf

```diff
@@ -1,18 +1,27 @@
 <%--
-  ~ Copyright Siemens AG, 2013-2016. Part of the SW360 Portal Project.
+  ~ Copyright Siemens AG, 2013-2017. Part of the SW360 Portal Project.
   ~
   ~ All rights reserved. This program and the accompanying materials
   ~ are made available under the terms of the Eclipse Public License v1.0
   ~ which accompanies this distribution, and is available at
   ~ http://www.eclipse.org/legal/epl-v10.html
   --%>
+
 <div id="content">
     <div class="container-fluid">
         <div id="myTab" class="row-fluid">
             <ul class="nav nav-tabs span2">
                 <li <core_rt:if test="${selectedTab == 'Summary' || empty selectedTab}"> class="active" </core_rt:if> id="Summary" >    <a href="#tab-Summary">Summary</a></li>
                 <li <core_rt:if test="${selectedTab == 'Linked Releases'}">              class="active" </core_rt:if>  >                <a href="#tab-linkedReleases">Linked Releases</a></li>
                 <li <core_rt:if test="${selectedTab == 'Clearing'}">                     class="active" </core_rt:if> id="Clearing" >   <a href="#tab-ClearingDetails">Clearing Details</a></li>
+                <li <core_rt:if test="${selectedTab == 'ECC Details'}">                  class="active" </core_rt:if> id="ECC-Details"> <a href="#tab-ECCDetails">ECC Details
+                    <div id="eccStatusDiv"
+                        <core_rt:if test="${release.eccInformation.eccStatus.value == 0 || release.eccInformation.eccStatus.value == 3}"> class="notificationBulletSpan backgroundAlert" </core_rt:if> <%--ECCStatus.OPEN || ECCStatus.REJECTED--%>
+                        <core_rt:if test="${release.eccInformation.eccStatus.value == 1}"> class="notificationBulletSpan backgroundWarning" </core_rt:if> <%--ECCStatus.IN_PROGRESS--%>
+                        <core_rt:if test="${release.eccInformation.eccStatus.value == 2}"> class="notificationBulletSpan backgroundOK" </core_rt:if>> <%--ECCStatus.APPROVED--%>
```

Comment:
I tried it. It looks ok, but I'm not sure I want to change it. I think green color by itself is a sufficiently universal indication that it's "OK". If we add text for this status, then we have to think about "In Progress" and "Open", too. And that may just make matters more complicated without bringing clarity for end-user. So, I'd say let's leave it as is.
@mcjaeger, @bs-jokri, what's your take on this? Do you have any preferences?